src/main/java/frc/robot/RobotContainer.java

```diff
@@ -200,14 +201,17 @@ private void configureBindings() {
         //However, autobalancing is the last thing we do in autonomous, so that probably wont be necessary.
         //Even though it's a repeating command, all autonomous commands should automatically be canceled once teleop is enabled.
         new RepeatCommand(
-          new AutoBalance(m_chassisSubsystem, ahrs, kInitialPitchOffset).withTimeout(0.5)
-          .andThen(new WaitCommand(0.5))
+          new AutoBalance(m_chassisSubsystem, ahrs, kInitialPitchOffset).withTimeout(0.7)
```

Comment:
We should comment this out if not using for comp, hate to have some dangling code that could trip up drivers